### PR TITLE
[Bugfix:InstructorUI] Cant use Notebook Builder in Gradeables

### PR DESCRIPTION
**Pull request by chenjohnny-cs**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)

### What is the current behavior?
Notebook builder is completely unable to be accessed whether no matter what upload configuration you choose. It should not be that way, and you should not be able to modify PROVIDED folder.

### What is the new behavior?
Notebook Builder cannot access the PROVIDED folder through edit, but is able to create a new notebook builder anywhere into the UPLOADED folder.

### Other information?
Linked to https://github.com/Submitty/Submitty/pull/5945

### CODE REVIEW

File: site/public/js/notebook_builder/widgets/widget.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* exported NUM_MARKDOWN, Widget */
 
-const NUM_MARKDOWN = 0;
+var NUM_MARKDOWN = 0;
```
Comment (ziesski) [inline site/public/js/notebook_builder/widgets/widget.js:3]: This fails our js-lint check, could you explain why we need to change it from const to var?

**Review by ziesski — CHANGES_REQUESTED** [review]

File: site/public/js/notebook_builder/widgets/widget.js (hunk 1 above)
Comment (chenjohnny-cs) [reply site/public/js/notebook_builder/widgets/widget.js:3]: The markdown-widget.js is changing the value on NUM_MARKDOWN each time a markdown is created, with a const variable I believe that you can't change that value

I don't know if the export at the top changes something with the variable

Comment (chenjohnny-cs) [reply site/public/js/notebook_builder/widgets/widget.js]: I updated the variable from var to let to pass the js-lint check.

**Review by RayReyRam — APPROVED** [review]
Previously we talked about being unable to get into the notebook builder, this fixes it, you are able to enter the notebook builder without any issues. The code looks good, not much changes, cant seem to break anything and it passes all the lint test.

**Review by JoJo-ESC — COMMENTED** [review]
Code looks good changes are seen and works as expected.

**Review by JoJo-ESC — APPROVED** [review]

**Review by ziesski — APPROVED** [review]
Code looks good to me, haven't tested the functionality.

**Review by JoJo-ESC — APPROVED** [review]
Changes look good tested functionality related to new builder as well as provision edit permissions and expected behavior was shown.

**Review by williamjallen — APPROVED** [review]
Code looks reasonable.  Did not test manually.

**Pull request merged**